hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/TokenAccountBalanceMigrationTest.java

```diff
@@ -54,8 +56,9 @@ class TokenAccountBalanceMigrationTest extends IntegrationTest {
     private final TokenBalanceRepository tokenBalanceRepository;
     private final TokenAccountBalanceMigration tokenAccountBalanceMigration;
     private final TokenTransferRepository tokenTransferRepository;
+    private final TransactionTemplate transactionTemplate;
 
-    private AccountBalanceFile accountBalanceFile;
+    private AccountBalanceFile accountBalanceFile1, accountBalanceFile2;
```

Comment:
Our standard is to always declare on a separate line.

```suggestion
    private AccountBalanceFile accountBalanceFile1;
    private AccountBalanceFile accountBalanceFile2;
```